jsapp/js/dataInterface.ts

```diff
@@ -627,7 +627,7 @@ export interface AccountResponse {
   email: string;
   server_time: string;
   date_joined: string;
-  projects_url: string;
+  projects_url: string; // TODO: no longer used, kept here for response typing
```

Comment:
I feel like keeping this as TODO would make us think we would like to change this, and most probably we wouldn't - unless Back end stops sending it in the response.

Let's change it to a descriptive comment (the JSDoc one in line above it) stating that this is a link to a legacy view containing a list of projects.